grasp generation

bostoncleek opened the pull request:
Add grasp pose synthesis to current projects.

AndyZe commented on documentation/contributing/future_projects/index.markdown:
```diff
@@ -56,6 +57,16 @@ Feel free to contact [PickNik Robotics](https://picknik.ai/connect/) for further
+- **Programming skills**: C++, Python
+- **Difficulty**: Medium
+- **Potential mentors**: Mark Moll
+- **Description**: Improve grasp pose synthesis within MoveIt and the MoveIt Task constructor. Current grasp sythesis algorithms pair deep neural networks and sampling point clouds, see [GPD](https://github.com/atenpas/gpd) and [Dex-Net](https://berkeleyautomation.github.io/dex-net/). An intial effort using the previous grasping methods includes a [demo](https://github.com/PickNikRobotics/deep_gr ...
```
Spell check:  `synthesis`, `initial`

AndyZe commented:
I'll give it a day to see if anybody else wants to comment, then merge.  :+1:

AndyZe commented:
It looks like you'll have to fix these links before I can merge. Sorry! They weren't caused by your PR:

> - ./_site/moveit!/ros/2013/05/06/icra-roscon-trip-report.html
> 
>   *  External link http://www.willowgarage.com/blog/2013/05/06/moveit failed: 404 No error
> 
>   *  External link http://www.willowgarage.com/blog/2013/05/12/moveit-running-baxter-research-robot-1 failed: 404 No error

bostoncleek commented:
> It looks like you'll have to fix these links before I can merge. Sorry! They weren't caused by your PR:
> 
> > * ./_site/moveit!/ros/2013/05/06/icra-roscon-trip-report.html
> >   
> >   * External link http://www.willowgarage.com/blog/2013/05/06/moveit failed: 404 No error
> >   * External link http://www.willowgarage.com/blog/2013/05/12/moveit-running-baxter-research-robot-1 failed: 404 No error
> 
> If the links are badly deprecated (looks like they're from 2013) you could probably delete them

I replaced one link and deleted the other.

v4hn commented on documentation/contributing/future_projects/index.markdown:
```diff
@@ -171,4 +182,4 @@ For this, we need to identify and implement standard tasks at varying levels of
 - **Potential mentors**: Mark Moll, David Lu!!
-- **Description**: The goal is to provide a standard way for integrating mobile base into move group that typically handles arms. While there's a non-standard way to incorporate holonomic drive that requires modifying your robot model ([example](https://groups.google.com/forum/#%21searchin/moveit-users/virtual$20base/moveit-users/G7vG7r23YzI/vpMUoa1hza4J)), customizing your robot package for such ...
\ No newline at end of file
+- **Description**: The goal is to provide a standard way for integrating mobile base into move group that typically handles arms. While there's a non-standard way to incorporate holonomic drive that requires modifying your robot model ([example](https://groups.google.com/forum/#%21searchin/moveit-users/virtual$20base/moveit-users/G7vG7r23YzI/vpMUoa1hza4J)), customizing your robot package for such ...
```
I'm at a loss trying to find a diff in this line. Did you change the line ending?

bostoncleek replied:
I did not change anything here.

v4hn commented on documentation/contributing/future_projects/index.markdown:
```diff
@@ -56,6 +57,16 @@ Feel free to contact [PickNik Robotics](https://picknik.ai/connect/) for further
+- **Programming skills**: C++, Python
+- **Difficulty**: Medium
+- **Potential mentors**: Mark Moll
+- **Description**: Improve grasp pose synthesis within MoveIt and the MoveIt Task constructor. Current grasp synthesis algorithms pair deep neural networks and sampling point clouds, see [GPD](https://github.com/atenpas/gpd) and [Dex-Net](https://berkeleyautomation.github.io/dex-net/). An initial effort using the previous grasping methods includes a [demo](https://github.com/PickNikRobotics/deep_ ...
```
I guess this says you won't have the time anymore to address this yourself?

bostoncleek replied:
I am working right now to finish the changes Robert requested and hopefully will get that PR merged within a couple weeks.
